integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/mutation/DeleteExecution.scala

```diff
@@ -58,14 +58,15 @@ object DeleteExecution {
       dataRdd: RDD[Row],
       timestamp: String,
       isUpdateOperation: Boolean,
-      executorErrors: ExecutionErrors): Boolean = {
+      executorErrors: ExecutionErrors): (Boolean, Seq[String]) = {
```

Comment:
It seems the first return value is always true
And it is hard to understand this function, can you add comment for it

Reply:
ok updated